### DISCUSSION TITLE
Where do the files go?

### DISCUSSION THREAD

**Discussion by targetlucked69** [crazy-max/ghaction-xgo#292] [Q&A] [answered]
Hi, where do the files actually go? I have set up my github action and on push it builds successfully and the target folder is `build` (default) but I don't see any binary file somewhere on my repository? Should I run this setup locally on my PC?

I am using M1 Mac Pro locally.

Thank you for this!

**Comment by slashtechno** [comment]
I suspect it may be `/build`. I couldn't find it either initially. I suggest you set your destination to `/home/runner/work/repo-name/builds`. At least that's how I did it, and it's been working perfectly.

If you're looking to upload the file as an artifact, you can use [actions/upload-artifact](https://github.com/actions/upload-artifact). If you're looking to publish a release, I suggest [softprops/action-gh-release](https://github.com/softprops/action-gh-release).

**Comment by crazy-max** [comment, marked as answer]
What's defined by [`dest` input](https://github.com/crazy-max/ghaction-xgo#inputs) (defaults to `build` in your workspace)
